Code/C++/Brian_Kernighan.cpp

```diff
@@ -60,8 +58,8 @@ int main(){
     2) input   : 39
        output  : 4
 
-        TIME COMPLEXITY -> O(No. of set bits in input)
+        TIME COMPLEXITY -> O(n) where n is No. of set bits in input.
         
         SPACE COMPLEXITY -> O(1)
```

Comment:
lowercase!

Reply:
I have made the changes that were asked. Please review it and let me know if any other changes are required. I am open to learning. If it is okay I would request you to merge this.  
Regards,
Debanjan Poddar
@rudrakshi99 
@tarun26091999 
@plazzy99 
@raksha009